src/profile.ts

```diff
@@ -98,6 +98,13 @@ export class Profile {
     return config;
   }
 
+  // Returns the corresponding registration form to edit a user profile.
+  // We are not interested in a non-existant registry link, since we check before hand if the name exists.
+  registry(config: Config): string {
+    if (this.profile?.ens) return `/registrations/${parseEnsLabel(this.profile.ens.name, config)}`;
+    else return `https://self.id/${formatCAIP10Address(this.profile.address, "eip155", config.network.chainId)}`;
```

Comment:
Could we keep `self.id` in `config.json`? Perhaps under `ceramic.registry`?

Reply:
Yeah sure :+1:

Reply:
Done @cloudhead